components/main-chef-wrapper/cmd/capture.go

```diff
@@ -36,6 +36,7 @@ var (
 Captures a node's state as a local chef-repo, which can then be used to
 converge locally.
 `,
+		DisableFlagParsing: true,
```

Comment:
Good find with disableFlagParsing; but I would expect that we specify this only on commands that accept arguments that we can't reasonably shadow in our wrappers. It seems like we're using it in more places than that?

Reply:
Yes I agree with this, but if I am not specifying this to the commands like `clean-policy-cookbooks` which don't accept the arguments than when I am passing any invalid flags to these command  these flags are not getting parsed as an argument and the cobra tries to find the invalid flag here itself without letting it to invocate chef-cli and exits the OS on the Execute function without any error.